code/modules/clothing/head/head.dm

```diff
@@ -6,12 +6,6 @@
 	w_class = WEIGHT_CLASS_SMALL
 	var/anti_hug = 0
 
-/obj/item/clothing/head/update_clothing_icon()
```

Comment:
Is this intentional?

Reply:
surprisingly, didn't seem to affect anything.